cxx-qt-lib/src/qcolor.rs

```diff
@@ -60,153 +60,175 @@ pub struct QColor {
     _pinned: PhantomData<PhantomPinned>,
 }
 
-// TODO: figure out how to make Syntax and Example compile as code
-// and then change ```ignore back to ```
-
-/// Construct a QColor on the Rust stack.
-///
-/// # Syntax
-///
-/// In statement position:
-///
-/// ```ignore
-/// # use cxx_qt_lib::{let_qcolor, Color};
-/// # let expression = Color::ARGB { alpha: 0, red: 255, green: 0, blue: 0 });
-/// let_qcolor!(var = expression);
-/// ```
-///
-///
-/// The `expression` must refer to a `&cxx_qt_lib::Color`.
-///
-/// The macro expands to something resembling `let $var: Pin<&mut QColor> =
-/// /*???*/;`. The resulting [`Pin`] can be deref'd to `&QColor` as needed.
-///
-/// # Example
-///
-/// ```ignore
-/// use cxx_qt_lib::{let_qcolor, QColor};
-///
-///
-/// fn f(s: &QColor) {/* ... */}
-///
-/// fn main() {
-///     let_qcolor!(s = "example");
-///     f(&s);
-/// }
-/// ```
-#[macro_export]
-macro_rules! let_qcolor {
-    ($var:ident = $value:expr $(,)?) => {
-        let mut stack_qcolor = $crate::private::StackQColor::new();
-        #[allow(unused_mut, unused_unsafe)]
-        let mut $var = match $value {
-            let_qcolor => unsafe { stack_qcolor.init(let_qcolor) },
-        };
-    };
-}
-
 impl QColor {
     /// `QColor` is not constructible via `new`.
-    /// Instead, use the [`let_qcolor!`] macro.
+    /// Instead, use [`Color::from_rgba`].
+    //
+    // FIXME: Do we still need this?
```

Comment:
I think the "new" function here can be removed.
You should never be able to call it, so it shouldn't exist.

Reply:
Right, and it will allow us to remove the Private struct in the lib.rs in the next pull request, done :-)